cmd/osctl/cmd/kubeconfig.go

```diff
@@ -56,11 +60,19 @@ Kubeconfig will be written to PWD/kubeconfig or [local-path]/kubeconfig if speci
 				localPath = args[0]
 			}
 
+			//Drop the existing kubeconfig before writing the new one if force flag is specified.
+			if force {
+				err = os.Remove(filepath.Clean(localPath + "/kubeconfig"))
```

Comment:
better to use `filepath.Join`

Reply:
fixed this up a bit to clean up the localpath and then use filepath.Join if force is supplied.